packages/orbit-components/src/Switch/__tests__/index.test.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+import { screen, render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Switch from "..";
+
+describe("Switch", () => {
+  it("should have expected DOM output", () => {
+    const dataTest = "test";
+    const onChange = jest.fn();
+    const onFocus = jest.fn();
+
+    render(<Switch dataTest={dataTest} checked onChange={onChange} onFocus={onFocus} />);
+
+    userEvent.click(screen.getByRole("button"));
+    expect(onChange).toHaveBeenCalled();
+
+    userEvent.tab();
+
+    expect(screen.getByRole("switch")).toHaveAttribute("checked");
+    expect(onFocus).toHaveBeenCalled();
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+  });
+
+  it("should be disabled", () => {
+    const onChange = jest.fn();
+    render(<Switch checked onChange={onChange} disabled />);
+
+    userEvent.click(screen.getByRole("button"));
+    expect(onChange).not.toHaveBeenCalled();
+    expect(screen.getByRole("switch")).toHaveAttribute("disabled");
+  });
+});
```

Comment:
Pressing <kbd>⮐</kbd> no longer works, could add a test for it as well as space bar?

Reply:
fixed